tests/phpcs.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
```

Comment:
It is probably better to use the official Marketplace acceptance rule set which can be found here:
https://github.com/magento/marketplace-eqp/blob/master/MEQP1/ruleset.xml

We would then be interested in severity 10 level errors run as such
`vendor/bin/phpcs /path/to/your/extension --standard=MEQP1 --severity=10 --extensions=php,phtml`

perhaps even better if this was downloaded live by CI so as to make sure that we keep up to date, but if not, the above might be a good choice.

Reply:
This lite config file already show a lot of errors. We could replace it later with more strict

Reply:
@oldDriver 
The point I wish to communicate is that our plugin is already in the Magento Marketplace
https://marketplace.magento.com/bolt-bolt.html

It is not optional that we pass the ruleset.xml listed above, it is mandatory that we pass that ruleset at severity level 10.  This is the only critically relevant code sniffing configuration for the M1 plugin.